pgjdbc/src/main/java/org/postgresql/sspi/SSPIClient.java

```diff
@@ -88,17 +88,17 @@ public boolean isSSPISupported() {
        * won't have JNA and this will throw a NoClassDefFoundError.
        */
       if (!Platform.isWindows()) {
-        LOGGER.log(Level.WARNING, "SSPI not supported: non-Windows host");
+        LOGGER.log(Level.FINE, "SSPI not supported: non-Windows host");
         return false;
       }
       /* Waffle must be on the CLASSPATH */
       Class.forName("waffle.windows.auth.impl.WindowsSecurityContextImpl");
       return true;
     } catch (NoClassDefFoundError ex) {
-      LOGGER.log(Level.WARNING, "SSPI unavailable (no Waffle/JNA libraries?)", ex);
+      LOGGER.log(Level.FINE, "SSPI unavailable (no Waffle/JNA libraries?)", ex);
```

Comment:
-1 as it might highlight configuration issues.
On the other hand, it could make sense to make this one-time-only log message (e.g. `static boolean sspiUnavailable` in `SSPIClient` class